SmartDeviceLinkTests/RPCSpecs/EnumSpecs/SDLAudioStreamingIndicatorSpec.m

```diff
@@ -0,0 +1,24 @@
+//
+//  SDLAudioStreamingIndicatorSpec.m
+//  SmartDeviceLink
+
+
+#import <Foundation/Foundation.h>
+
+#import <Quick/Quick.h>
+#import <Nimble/Nimble.h>
+
+#import "SDLAudioStreamingIndicator.h"
+
+QuickSpecBegin(SDLAmbientLightStatusSpec)
```

Comment:
Test cases are not compiling because of a duplicate test case name. This test case should be named `QuickSpecBegin( SDLAudioStreamingIndicatorSpec)`.